doc/source/api.rst

```diff
@@ -449,109 +449,103 @@ Datetimelike Properties
 
 ``Series.dt`` can be used to access the values of the series as
 datetimelike and return several properties.
-Due to implementation details the methods show up here as methods of the
-``DatetimeProperties/PeriodProperties/TimedeltaProperties`` classes. These can be accessed like ``Series.dt.<property>``.
```

Comment:
I was thinking, should we keep this sentence (but eg between brackets as a note, that they are implemented like that, as users can still see this if they do `s.str?` or `type(s.str)`)?